duden/locale/de_DE.po

```diff
@@ -0,0 +1,112 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR ORGANIZATION
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2018-05-18 11:12+0200\n"
+"PO-Revision-Date: 2018-05-18 13:39+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: main.py:79
+msgid "Word type: "
+msgstr "Wortart: "
+
+#: main.py:81
+msgid "Usage: "
+msgstr "Nutzung: "
+
+#: main.py:82
+msgid "Commonness: {}/5"
+msgstr "Häufigkeit: {}/5"
+
+#: main.py:84
+msgid "Separation: "
+msgstr "Worttrennung: "
+
+#: main.py:87
+msgid "Meaning overview:"
+msgstr "Bedeutungsübersicht:"
+
+#: main.py:91
+msgid "Synonyms:"
+msgstr "Synonyme:"
+
+#: main.py:95
+msgid "Typical compounds:"
+msgstr "Typische Wortverbindungen:"
+
+#: main.py:431
+msgid "Connection could not be established. Check your internet connection."
+msgstr "Verbindung konnte nicht hergestellt werden. Prüfen Sie Ihrer Internetverbindung."
+
+#: main.py:442
+msgid "Unexpected HTTP response status code {}"
+msgstr "Unvorhersehbare HTTP-Response-Status-Code {}"
+
+#: main.py:480
+msgid "Display word and its article"
+msgstr "Wort und sein Artikel anzeigen"
+
+#: main.py:482
+msgid "Display the word itself"
+msgstr "Das Wort selbst anzeigen"
+
+#: main.py:484
+msgid "Display word article"
+msgstr "Wort Artikel anzeigen"
+
+#: main.py:486
+msgid "Display word's part of speech"
+msgstr "Wortaussprache anzeigen"
+
+#: main.py:488
+msgid "Display commonness of the word, on scale 1 to 5"
+msgstr "Häufigkeit des Worts, auf einer Skala von 1 bis 5, anzeigen"
+
+#: main.py:491
+msgid "Display the context in which the word is used"
+msgstr "Den Zusammenhang, in dem das Wort benutzt wird, anzeigen"
+
+#: main.py:494
+msgid "Display proper word separation, each part on a separate line."
+msgstr "Richtige Worttrennung, jeden Teil in einer eigenen Zeile, anzeigen."
+
+#: main.py:497
+msgid "Display word meaning overview"
+msgstr "Wortbedeutungsübersicht anzeigen"
+
+#: main.py:499
+msgid "List word synonyms, each on a separate line"
+msgstr "Wort Synonyme, jede in einer eigenen Zeile, listen"
+
+#: main.py:501
+msgid "Display word origin"
+msgstr "Wortherkunft anzeigen"
+
+#: main.py:503
+msgid "List common word compounds"
+msgstr "Typische Wortverbindungen listen"
+
+#: main.py:505
+msgid "List grammar forms"
+msgstr "Grammatikformen listen"
```

Comment:
***?** **auf**listen*. What is meant by *grammar forms*?

Reply:
Dativ, Konjunktiv II, usw.

Reply:
So this refers to the "Grammatik" section (on duden.de)? If so, I'd say *Grammatik~formen~ ~listen~ **anzeigen***.